package.json

```diff
@@ -24,7 +25,8 @@
     "react": "^18",
     "react-dom": "^18",
     "react-icons": "^4.12.0",
-    "react-leaflet": "^4.2.1"
+    "react-leaflet": "^4.2.1",
+    "sitemap": "^7.1.1"
```

Comment:
generate the sitemap directly, dont need to install separate packages or make the function to generate it